CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- Change title that is shown in pages list.
+- Send partial route data to be saved.
```

Comment:
How about the added functionality of editing the title of pages without context?